src/querySet.js

```diff
@@ -107,6 +106,11 @@ const QuerySetRequest = stampit().compose(Request)
         return ResultSet(this, response, objects);
       }
 
+      if(this.endpoint === 'users') {
```

Comment:
this is no way to go... one custom `if` for one model in *core* part is a really bad idea :P